service/service.go

```diff
@@ -22,6 +22,7 @@ type Service struct {
 	Teams        []string
 	Doc          string
 	IsRestricted bool `bson:"is_restricted"`
+	Instances    []ServiceInstance
```

Comment:
cool, Andrews. I agree it's a better approach. :smile: 
Let's see if the guys have any other suggestions and I can start working on improving this PR.

Thanks for your feedback.